toolset/Documentation/project-docs/developer-guide.md

```diff
@@ -29,6 +29,8 @@ build.cmd
 
 The build script will output a `dotnet` installation to `artifacts\tmp\Debug\dotnet` that will include any local changes to the .NET Core CLI.
 
+To open the solution in Visual Studio, be sure to run `artifacts\sdk-build-env.bat` first and open with `devenv sdk.sln`.
```

Comment:
You first need to run the build script in order to produce the sdk-build-env.bat file.  The reason to open VS this way is to use the stage 0 SDK from the `.dotnet` folder.  It's not technically necessary if you have a globally-installed version of the .NET SDK with a high enough version.